### ISSUE TITLE
Error when following the user guide to get started

### ISSUE THREAD

**Issue by MattHJensen** [PSLmodels/Cost-of-Capital-Calculator#314]
I am following the [user guide to run ccc](https://github.com/PSLmodels/cost-of-capital-calculator#usingcontributing-to-cost-of-capital-calculator). After activating ccc-dev, I received the following error when attempting to run `python example.py`. Any tips or suggestions would be appreciated! My goal is to work from source. 

```

Matthews-MBP:Cost-of-Capital-Calculator matthewjensen$ conda activate ccc-dev
(ccc-dev) Matthews-MBP:Cost-of-Capital-Calculator matthewjensen$ subl example.py
(ccc-dev) Matthews-MBP:Cost-of-Capital-Calculator matthewjensen$ python example.py
Traceback (most recent call last):
  File "/Users/matthewjensen/Projects/Cost-of-Capital-Calculator/ccc/utils.py", line 122, in read_egg_csv
    pkg_resources.resource_stream(
  File "/Users/matthewjensen/anaconda3/envs/ccc-dev/lib/python3.8/site-packages/pkg_resources/__init__.py", line 1149, in resource_stream
    return get_provider(package_or_requirement).get_resource_stream(
  File "/Users/matthewjensen/anaconda3/envs/ccc-dev/lib/python3.8/site-packages/pkg_resources/__init__.py", line 356, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/Users/matthewjensen/anaconda3/envs/ccc-dev/lib/python3.8/site-packages/pkg_resources/__init__.py", line 899, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/matthewjensen/anaconda3/envs/ccc-dev/lib/python3.8/site-packages/pkg_resources/__init__.py", line 785, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'ccc' distribution was not found and is required by the application

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "example.py", line 28, in <module>
    assets = Assets()
  File "/Users/matthewjensen/Projects/Cost-of-Capital-Calculator/ccc/data.py", line 62, in __init__
    self._read_data(data)
  File "/Users/matthewjensen/Projects/Cost-of-Capital-Calculator/ccc/data.py", line 136, in _read_data
    assetdf = read_egg_csv(data)  # pragma: no cover
  File "/Users/matthewjensen/Projects/Cost-of-Capital-Calculator/ccc/utils.py", line 128, in read_egg_csv
    raise ValueError('could not read {} data from egg'.format(fname))
ValueError: could not read ccc_asset_data.csv data from egg

```

**Comment by jdebacker** [comment]
@MattHJensen  I was able to replicate this error.

Can you try to do a `pip install -e .` from the main directory of the repo before doing `python example.py` and report back?

Thanks!

**Comment by MattHJensen** [comment]
> Can you try to do a pip install -e . from the main directory of the repo before doing python example.py and report back?

Success!

**Comment by jdebacker** [comment]
Fixed via PR #317.
